requirements.txt

```diff
@@ -1,3 +1,4 @@
 beautifulsoup4==4.6.0
 PySocks==1.6.7
 stem==1.5.4
```

Comment:
Stem is no longer being used and request requirement should be added for version 2.18.4